src/js/pages/MyProfile/index.scss

```diff
@@ -22,4 +23,20 @@
   z-index: 80;
   position: relative;
   left: -10px;
+}
+
+.status-green {
+  background-color: $green;
+}
+
+.status-gray {
+  background-color: $dark;
+}
+
+.status-blue {
+  background-color: $blue;
+}
+
+.status-red {
+  background-color: $red;
```

Comment:
Sorry to make you make you do stuff then erase, but bootstrap already has classes for that. So instead of creating `status-color` classes, you can just use the `.bg-color` classes: https://getbootstrap.com/docs/4.3/utilities/colors/#background-color

Again,
red = danger
green = success
blue = primary
dark = dark (lucky) (could be secondary depending on your choice though)